pom.xml

```diff
@@ -29,18 +29,23 @@
     <dependencies>
         <dependency>
             <groupId>io.grpc</groupId>
-            <artifactId>grpc-netty</artifactId>
```

Comment:
I updated these deps as per
https://github.com/grpc/grpc-java/blob/master/SECURITY.md#troubleshooting

so it doesn't matter what version of netty is being used